### ISSUE TITLE
Extension crashing - requiring reload

### ISSUE THREAD

**Issue by Naxsun** [tallycash/extension#2698]
### Discord Discussion Link

na

### What browsers are you seeing the problem on?

Chrome

### What were you trying to do?

Use the extension

### What did not work?

Now and then the extension crashes, and requires a reload. Cases I'm aware off so far:
- Random in time, a pop-up message on the top right appears to inform the extension has crashed
- After an extension version update
- Shortly after re-starting my laptop.

In all cases the extension wasn't actually used, it was running in the background.

![image](https://user-images.githubusercontent.com/71384922/204781260-585b8237-04cf-4b93-84df-359620d59ed6.png)

### Version

v0.18.2

**Comment by greg-nagy** [comment]
We will need help to reproduce this properly, but it's bad, so added it to the current sprint

**Comment by Naxsun** [comment]
It happens randomly at the moment and haven't been able to trigger it. Is there anything I can record at the moment it happens?

**Comment by Naxsun** [comment]
Tried another restart to see if it triggered the extension crashing again and it did. Again ~ 5 minutes after the restart. Below is a snapshot of the error pop-over that comes up.

<img width="308" alt="Screen Shot 2022-12-01 at 08 45 23" src="https://user-images.githubusercontent.com/71384922/204995492-aa06b5e8-11cd-415b-a6b6-e9ebe705b697.png">

**Comment by greg-nagy** [comment]
@Naxsun should we remove the hight severity from this? Feels very occasional and rare

**Comment by Naxsun** [comment]
Agreed! I had to remove some accounts to speed it up, and it hasn't crashed for me ever since. Maybe we take it as a known fact for now, and we can judge later if it's something we'd need to fix later on.
